webpack.config.js

```diff
@@ -1,16 +1,9 @@
 var path = require('path')
 
-var pkg = require('./package.json')
-var tags = makeTags(pkg.version)
-
-var entry = {}
-entry['cf-extension-api'] = './lib/api/index.js'
-tags.forEach((tag) => {
-  entry[`tagged/cf-extension-api-v${tag}`] = './lib/api/index.js'
-})
-
 module.exports = {
-  entry: entry,
+  entry: {
+    'cf-extension-api': './lib/api/index.js'
```

Comment:
We want the output file to be named `cf-extension-api.js`. It would be named `main.js` if we followed your approach. We could control this with the `output.filename` configuration but I think this is not he right place: If we want to add mulptile targets we would need to revert again.